server/tests/test_notebook_view.py

```diff
@@ -31,6 +33,7 @@ def test_notebook_view(client, test_notebook):
 
 @pytest.mark.parametrize("logged_in", [True, False])
 def test_new_notebook_view(client, fake_user, logged_in):
+    random.seed(0)
```

Comment:
I searched a bit for a pytest fixture that would do this for me, but didn't see anything, at least nothing simple.